src/probe/probe.c

```diff
@@ -800,6 +800,10 @@ static void probe_cb_produce(void *arg, enum notify_id type, void *data)
 			if (ret < 0)
 				goto err;
 		}
+		/* check if more than 75% of buffer size is already used */
+		if (_probe->ext_dma.dmapb.size - _probe->ext_dma.dmapb.avail <
+		    _probe->ext_dma.dmapb.size >> 2)
```

Comment:
I still feel like the comment and code have their logic inverted to one another. 75% used means 25% available, yet you check for 25% of size-available (=used).

Reply:
avail means available data, not free so size-available = free

Reply:
Ah okay.